e2etests/testcase_test.go

```diff
@@ -1990,7 +1990,7 @@ var testCases = []testCase{
 				},
 				AddedBy:  "sha256:36e8e9714b9a509fae9e515ff16237928c3d809f5ae228b14d2f7d7605c02623",
 				Location: "jars/jackson-databind-2.9.10.4.jar",
-				FixedBy:  "2.12.6.1",
+				FixedBy:  "2.14.0",
```

Comment:
This should really be done in a separate PR, but it's fine for now

Reply:
Yeah, I was using another pr but the CI always timed out. Meanwhile I didn't remove this change from this pr.